src/plugins/formActions/EditPropertiesAction.js

```diff
@@ -92,6 +99,12 @@ export default class EditPropertiesAction extends PropertiesAction {
         const createWizard = new CreateWizard(this.openmct, this.domainObject, objectPath[1]);
         const formStructure = createWizard.getFormStructure(false);
         formStructure.title = 'Edit ' + this.domainObject.name;
+        //If we're in edit mode AND want to edit properties for the same domain object
+        //In this case, saving will put the object in view-only mode
+        //TODO: Maybe we should block editing properties if someone is in Edit mode?
+        if (!this.openmct.editor.isEditing()) {
```

Comment:
This is changing the current behavior in that pressing 'Save' or 'Cancel' on an Edit Properties window while in Edit mode will now complete the edit (previously it did not).

There is one odd case I've found where entering into the "Edit Properties" window while not in edit mode, and pressing the "X" to close the window will leave edit mode active instead of cancelling it. Not sure if this has a negative impact, but could be surprising to a user.

Reply:
Yes, we need a follow up task to not allow editing properties if the object is in edit mode already.

I've resolved the bug where canceling the edit properties action leaves it in edit mode.